copier.yml

```diff
@@ -1,5 +1,3 @@
-_exclude:
```

Comment:
Is this by intention?

Reply:
@huxuan  Yep, it is by intention! Because I need to create a directory `includes` for mkdocs pymdownx.snippets。

Based on copier [importing-jinja-templates-and-macros](https://copier.readthedocs.io/en/stable/configuring/#importing-jinja-templates-and-macros), use _exclude or _subdirectory is the same. Moreover, [it is []](https://copier.readthedocs.io/en/stable/configuring/#exclude) for _exclude when use a _subdirectory. So it is redundant basically before.